app/models/role.rb

```diff
@@ -335,6 +334,13 @@ def self.permissions_with_descriptions
           'Client Extras',
         ],
       },
+      can_generate_homeless_verification_pdfs: {
+        description: '',
```

Comment:
This needs a description.

Reply:
Maybe: Allows access to generate and view homeless verification PDFs for any client the user has access to.

Reply:
🤦 ✅